README.md

```diff
@@ -65,7 +75,8 @@ In your Style-Dictionary config, you can **either** use the `tokens-studio` tran
   "source": ["**/*.tokens.json"],
   "platforms": {
     "css": {
-      "transformGroup": "tokens-studio",
+       // Need to add `custom/tokens-studio` if you added custom transformers
+      "transformGroup": "custom/tokens-studio",
```

Comment:
- Comments in JSON are invalid
- Custom Transform Group example is further down the README already, this example should show the default scenario which is by just using "tokens-studio" transformGroup